docs/tern.shapes.ttl

```diff
@@ -1066,7 +1066,7 @@ tern-shapes:Result-isResultOf
     skos:prefLabel "is result of" ;
     sh:description "Relation linking a Result to the Observation or Actuation or act of Sampling that created or caused it." ;
     sh:maxCount 1 ;
-    sh:message "A `tern:Result` _MUST_ have exactly 1 `sosa:isResultOf` predicate where the value node is an IRI of type `tern:Observation` or `tern:Sampling`." ;
+    sh:message "A `tern:Result` _MAY_ have at most 1 `sosa:isResultOf` predicate where the value node is an IRI of type `tern:Observation` or `tern:Sampling`." ;
```

Comment:
Hi @edmondchuc , if a result can be from multiple activities, is it `at least 1` instead of `at most 1`?

Reply:
It is `sh:maxCount 1`, hence the `at most 1`. It is a `_MAY_` because it can either be `0..1`.

With the TERN Ontology, we currently assume results are from one activity.

> result can be from multiple activities

Can you provide an example?

Reply:
It's from this comment? https://github.com/ternaustralia/ontology_tern/pull/193#discussion_r890730290

Reply:
I don't see a need to do this. It is better we don't allow this for now unless there is consensus, to keep complexity low. 

By providing a relationship indicating a result can be from one or more activities, we need to introduce additional relationships to indicate in the data which activity was the proximate activity and which activities are transitive activities. This is similar to why the relationship `sosa:hasUltimateFeatureOfInterest` was added to relate things to the ultimate feature of interest.

If there is an actual need to allow for results to come from more than one activity, then we can add this in the future and properly document it.